### PR TITLE
doc(website): Slack plugin blog post

### PR DESCRIPTION
**Pull request by hermanschaaf**
Blog post for when the Slack plugin in https://github.com/cloudquery/cloudquery/pull/5624 is released

### CODE REVIEW

File: website/pages/blog/introducing-the-slack-source-plugin.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+CloudQuery is an open-source, extendable framework that gives you a single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are happy to announce the release of the Slack source plugin for CloudQuery.
+
+Slack is a popular communication tool used by many organizations. With the CloudQuery Slack source plugin, you can now load Slack workspace data into Postgres, Snowflake, BigQuery, SQLite, or any other destination supported by CloudQuery. 
+
+As part of its initial release, the Slack plugin supports pulling data for the following Slack resources:
+ - users into the [slack_users](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/docs/tables/slack_users.md) and [slack_user_presences](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/docs/tables/slack_user_presences.md) tables,
```
Comment (erezrokah) [inline website/pages/blog/introducing-the-slack-source-plugin.md:21]:
```suggestion
 - users into the [`slack_users`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/docs/tables/slack_users.md) and [slack_user_presences](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/docs/tables/slack_user_presences.md) tables,
```

Quoting the link text should fix these

File: website/pages/blog/introducing-the-slack-source-plugin.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+CloudQuery is an open-source, extendable framework that gives you a single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are happy to announce the release of the Slack source plugin for CloudQuery.
+
+Slack is a popular communication tool used by many organizations. With the CloudQuery Slack source plugin, you can now load Slack workspace data into Postgres, Snowflake, BigQuery, SQLite, or any other destination supported by CloudQuery. 
+
+As part of its initial release, the Slack plugin supports pulling data for the following Slack resources:
+ - users into the [`slack_users`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/docs/tables/slack_users.md) and [slack_user_presences](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/docs/tables/slack_user_presences.md) tables,
```
Comment (hermanschaaf) [inline website/pages/blog/introducing-the-slack-source-plugin.md:21]:
```suggestion
 - users into the [`slack_users`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/docs/tables/slack_users.md) and [`slack_user_presences`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/docs/tables/slack_user_presences.md) tables,
```

File: website/pages/blog/introducing-the-slack-source-plugin.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+ - teams into the [`slack_teams`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/docs/tables/slack_teams.md) table,
+ - channels into the [`slack_conversations`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/docs/tables/slack_conversations.md) table,
+ - messages into the [`slack_conversation_histories`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/docs/tables/slack_conversation_histories.md) and [slack_conversation_replies](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/docs/tables/slack_conversation_replies.md) tables,
```
Comment (hermanschaaf) [inline website/pages/blog/introducing-the-slack-source-plugin.md:24]:
```suggestion
 - messages into the [`slack_conversation_histories`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/docs/tables/slack_conversation_histories.md) and [`slack_conversation_replies`](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/slack/docs/tables/slack_conversation_replies.md) tables,
```

File: website/pages/blog/introducing-the-slack-source-plugin.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+  single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are
+  happy to announce the release of the Slack source plugin.
+author: hermanschaaf
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+CloudQuery is an open-source, extendable framework that gives you a single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are happy to announce the release of the Slack source plugin for CloudQuery.
```
Comment (disq) [inline website/pages/blog/introducing-the-slack-source-plugin.md:16]:
```suggestion
CloudQuery is an open-source, extendable framework that gives you a single-pane-of-glass to your cloud-infrastructure. Today, we are happy to announce the release of the Slack source plugin for CloudQuery.
```

The SQL part was removed in some of the recent posts (vercel)

Comment (yevgenypats) [inline website/pages/blog/introducing-the-slack-source-plugin.md:16]: Can we use the same tagline as on the website?

**Review by yevgenypats — APPROVED** [review]
Nice! Love the cross plugin queries!

File: website/pages/blog/introducing-the-slack-source-plugin.md (hunk 4 above)
Comment (hermanschaaf) [reply website/pages/blog/introducing-the-slack-source-plugin.md:16]: @yevgenypats "CloudQuery is an open source high performance data integration platform designed for security and infrastructure teams." ?

Comment (hermanschaaf) [inline website/pages/blog/introducing-the-slack-source-plugin.md:16]:
```suggestion
CloudQuery is an open source high performance data integration platform designed for security and infrastructure teams. Today, we are happy to announce the release of the Slack source plugin for CloudQuery.
```

File: website/pages/blog/introducing-the-slack-source-plugin.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+order by day, count desc
+```
+
+## Getting Started
+
+To get started syncing Slack data, see the [Slack source plugin documentation](/docs/plugins/sources/slack/overview) for instructions.
+
+## What's next
+
+The Slack API comes with some strict [rate limits](https://api.slack.com/docs/rate-limits) that mean syncing messages from channels with a long history can take a long time. We are going to continue expanding the Slack source plugin, adding support for more resources. If you are interested in a specific Slack resource, feel free to raise an [issue on GitHub](https://github.com/cloudquery/cloudquery/issues). Or if you need some help to get started, join us over on [Discord](https://www.cloudquery.io/discord), we'd love to help.
```
Comment (hermanschaaf) [inline website/pages/blog/introducing-the-slack-source-plugin.md:124]:
```suggestion
The Slack API comes with some strict [rate limits](https://api.slack.com/docs/rate-limits) that mean syncing messages from channels with a long history can take a long time. We are [thinking about ways to address this](https://github.com/cloudquery/cloudquery/issues/5809), please upvote or comment on the Github issue if you are interested!

We are also going to continue expanding the Slack source plugin, adding support for more resources as they become available in the Slack API. If you are interested in a specific Slack resource, feel free to raise an [issue on GitHub](https://github.com/cloudquery/cloudquery/issues). Or if you need some help to get started, join us over on [Discord](https://www.cloudquery.io/discord), we'd love to help.
```

File: website/pages/blog/introducing-the-slack-source-plugin.md (hunk 4 above)
Comment (hermanschaaf) [reply website/pages/blog/introducing-the-slack-source-plugin.md]: Updated

File: website/pages/blog/introducing-the-slack-source-plugin.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
+---
+title: Introducing the Slack Source Plugin
+tag: announcement
+date: 2022/12/20
+description: >-
+  CloudQuery is an open-source, extensible framework that gives you a
+  single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are
+  happy to announce the release of the Slack source plugin.
```
Comment (hermanschaaf) [inline website/pages/blog/introducing-the-slack-source-plugin.md:8]:
```suggestion
  CloudQuery is an open source high performance data integration 
  platform designed for security and infrastructure teams. Today, we are
  happy to announce the release of the Slack source plugin.
```

**Review by disq — APPROVED** [review]

**Pull request merged**